perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_def_login_namespace_lifescienceid_username.java

```diff
@@ -4,24 +4,80 @@
 import cz.metacentrum.perun.core.api.AttributeDefinition;
 import cz.metacentrum.perun.core.api.AttributesManager;
 import cz.metacentrum.perun.core.api.User;
+import cz.metacentrum.perun.core.api.exceptions.AlreadyReservedLoginException;
 import cz.metacentrum.perun.core.api.exceptions.AttributeNotExistsException;
+import cz.metacentrum.perun.core.api.exceptions.ConsistencyErrorException;
 import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import cz.metacentrum.perun.core.api.exceptions.LoginIsAlreadyBlockedException;
 import cz.metacentrum.perun.core.api.exceptions.WrongAttributeAssignmentException;
 import cz.metacentrum.perun.core.api.exceptions.WrongAttributeValueException;
 import cz.metacentrum.perun.core.api.exceptions.WrongReferenceAttributeValueException;
-import cz.metacentrum.perun.core.blImpl.ModulesUtilsBlImpl;
 import cz.metacentrum.perun.core.impl.PerunSessionImpl;
 
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class urn_perun_user_attribute_def_def_login_namespace_lifescienceid_username extends urn_perun_user_attribute_def_def_login_namespace{
 	private final static String elixirUsername = "urn:perun:user:attribute-def:def:login-namespace:elixir";
 	private final static String bbmriUsername = "urn:perun:user:attribute-def:def:login-namespace:bbmri";
 
+	private static final Pattern startWithLetterPattern = Pattern.compile("^[A-Za-z].*$");
+	private static final Pattern onlyNumbersPattern = Pattern.compile("^[0-9]+$");
+
+
 	@Override
 	public void changedAttributeHook(PerunSessionImpl sess, User user, Attribute attribute) {
 		trySetAttribute(sess, user, attribute, elixirUsername);
 		trySetAttribute(sess, user, attribute, bbmriUsername);
 	}
 
+	@Override
+	public void checkAttributeSyntax(PerunSessionImpl sess, User user, Attribute attribute) throws WrongAttributeValueException {
+		super.checkAttributeSyntax(sess, user, attribute);
+
+		if (attribute.getValue() != null) {
```

Comment:
This may be a nitpick, but if you'd have reversed the condition, you wouldn't need this big of an if. 
```suggestion
		if (attribute.getValue() == null) {
		    return;
		}
```